protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -343,6 +343,7 @@ enum CoreOperatorType {
   IMAGE_SUB_SCAN = 52;
   SEQUENCE_SUB_SCAN = 53;
   PARTITION_LIMIT = 54;
+  RUNTIME_FILTER = 55;
```

Comment:
It would be OK to not generate c++ protobufs in this PR since the C++ client doesn't actually (need to) use these values. We can lump the protobuf regeneration with some other C++ client PR.